socket.yml

```diff
@@ -3,19 +3,20 @@ description: Amazon transcribe Syncano Socket
 long_description: |
   Visit link to know more about Amazon Transcribe automatic speech recognition
   https://docs.aws.amazon.com/transcribe/latest/dg/what-is-transcribe.html
-version: 0.0.3
+version: 0.0.1
```

Comment:
Why did you decrease the socket.yml version? It should be bumped automatically after successful test runs during Circle CI deployment

Reply:
Oh I see.